okapi/services/attrs/attribute-definitions.xml

```diff
@@ -57,11 +57,12 @@ Legend and field usage:
             (use <groundspeak ... name="NAME" .../>)
 
 Notes: 
-- In opencaching-pl implementation all attributes are either ON or OFF.
+- All OC implementations have attributes either ON or OFF.
   Inverse meaning is handled by a different attribute.
-- In OCDE implementation attributes have same ID and use:
+- In OCDE implements internally the additional `inc` flag for GS compatibility 
+  of it's GPX exports (for the representation of the inverse meaning of an attribute):
```

Comment:
Not only OCDE, but also OCPL and OKAPI implement the 'inc' flag for GPX exports.

Reply:
Improved notes on `inc` flag. Thank you.